.github/workflows/laravel.yml

```diff
@@ -0,0 +1,32 @@
+name: Laravel
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  laravel-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.3'
```

Comment:
shouldn't we increase the version [here](https://github.com/RonasIT/laravel-helpers/blob/master/composer.json#L13) as well?